README.md

```diff
@@ -45,6 +45,9 @@ Required SDK configuration parameters include:
 ### Access Token Refresh on Expire
 SDK FHIR requests check whether the access token is expired before the data endpoint call. By default, an expired token will refresh. To disable token refresh, set `token_refresh_on_expire` to `false`.
 
+### Expired Data Access Grant
```

Comment:
This is in a section about config parameters. This should go somewhere else (maybe a standalone new section? maybe it would fit into another section? I'm not sure) since it doesn't have anything to do with config parameters. Additionally, a sentence or two of context might be helpful here in addition to pointing them to the section of the developer documentation.

Reply:
done